documentation/docs/30-advanced/20-hooks.md

```diff
@@ -232,6 +233,37 @@ During development, if an error occurs because of a syntax error in your Svelte
 
 > Make sure that `handleError` _never_ throws an error
 
+## Universal hooks
+
+The following can be added to `src/hooks.js`. Universal hooks run on both server and client (not to be confused with shared hooks, which are environment-specific). 
+
+### reroute
+
+This function allows you to change which route get's rendered. By returning a different path from it, that new path will be used to determine the route that needs to be rendered. It will not change the URL in the browser's address bar, or the value of `event.url`. Only the route matching logic is affected.
+
+```js
+/// file: src/hooks.router.js
+// @errors: 2345
+// @errors: 2304
+
+/** @type {Record<string, string>} */
+const mapping = {
+	"/en/about": "/en/about",
+	"/de/ueber-uns": "/de/about",
+	"/fr/a-propos": "/fr/about",
+}
+
+/** @type {import('@sveltejs/kit').Reroute} */
+export function reroute({ url }) {
+	if(url.pathname in mapping) {
+		return mapping[url.pathname];
+	}
+}
+```
+
+> `rewriteUrl` is not applied to the returned URL
```

Comment:
The `lang` parameter will be correctly derived from the returned pathname.

Using `reroute` will _not_ change the contents of the browser's address bar, or the value of `event.url`.